src/models/Coversation.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from "mongoose";
+
+const ConversationSchema = new mongoose.Schema(
+  {
+    conversationID: Number,
+    conversationName: String,
+  },
+  {
+    timestamps: true,
```

Comment:
we no need to knw when is the timestamp of the Conversation i think
we just need to know on the message